HW_06_Task2/HW_06_Task2/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+
+namespace HW_06_Task2
+{
+    class Program
+    {
+        static char[] replaceMethod(char[] txt)
```

Comment:
fist letter in C# methods - Upper case:

`static char[] replaceMethod(char[] txt)`   => `static char[] ReplaceMethod(char[] txt)`